crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -768,6 +771,52 @@ impl<'w, 's> Commands<'w, 's> {
     ) -> EntityCommands {
         self.spawn(Observer::new(observer))
     }
+
+    /// Registers a system and returns a [`SystemId`] so it can later be called by [`World::run_system`].
+    ///
+    /// It's possible to register the same systems more than once, they'll be stored separately.
+    ///
+    /// This is different from adding systems to a [`Schedule`](crate::schedule::Schedule),
+    /// because the [`SystemId`] that is returned can be used anywhere in the [`World`] to run the associated system.
+    /// This allows for running systems in a pushed-based fashion.
+    /// Using a [`Schedule`](crate::schedule::Schedule) is still preferred for most cases
+    /// due to its better performance and ability to run non-conflicting systems simultaneously.
+    pub fn register_system<I: 'static, O: 'static, M, S: IntoSystem<I, O, M> + 'static>(
+        &mut self,
+        system: S,
+    ) -> SystemId<I, O> {
+        self.register_boxed_system(Box::new(IntoSystem::into_system(system)))
+    }
+
+    /// Similar to [`Self::register_system`], but allows passing in a [`BoxedSystem`].
+    ///
+    ///  This is useful if the [`IntoSystem`] implementor has already been turned into a
+    /// [`System`](crate::system::System) trait object and put in a [`Box`].
+    pub fn register_boxed_system<I: 'static, O: 'static>(
+        &mut self,
+        system: BoxedSystem<I, O>,
+    ) -> SystemId<I, O> {
+        SystemId {
+            entity: self
+                .spawn(RegisteredSystem {
+                    initialized: false,
+                    system,
+                })
+                .id(),
+            marker: std::marker::PhantomData,
+        }
+    }
+
+    /// Removes a registered system if it exists. Unlike [`World`] does not return the system.
```

Comment:
```suggestion
    /// Removes a registered system if it exists.
    ///
    /// Unlike [`World`], this does not return the system.
```

I would prefer splitting these sentences up into multiple lines, so that only the first is shown as a summary.